### PR TITLE
Update macro pod

### PR DESCRIPTION
**Pull request by pstaabp**
Working on #811 noticed that some of the macros didn't have POD, others were in bad shape. 

This is intended to be some cleanup to the POD in the macros with no changes to code.  This was just an initial pass through on some of the missing POD and some other cases.  I did fair bit of exchanging leading tabs in POD to four spaces, but not throughout the macros.

### CODE REVIEW

**Comment by drgrice1** [comment]
If you want to go further with this, you could run `podchecker` on the macros.  Doing so reveals more issues with the POD.  The most common is a warning about `=head?` without preceding higher level.

File: macros/contexts/contextInequalitiesAllowStrings.pl [hunk 1]
```diff
@@ -14,8 +14,8 @@ =head3 Allows string answers for the set of real numbers and the empty set
 
 =pod
```
Comment (drgrice1) [inline macros/contexts/contextInequalitiesAllowStrings.pl:15]: You should also deleted the `=pod` here.  That doesn't do anything after `=head2`.

Comment (drgrice1) [reply macros/contexts/contextInequalitiesAllowStrings.pl:15]: I also see an `=cut` statement immediately before this on line 11.  That is also unneeded.

Comment (drgrice1) [reply macros/contexts/contextInequalitiesAllowStrings.pl:15]: I ran `podchecker` on this and it says the `=head3` should be a `=head2` (I am paraphrasing).  Also the parts in `=head1 NAME` and `=head1 Synopsis` should have no indentation.  `podchecker` says "*** WARNING: Verbatim paragraph in NAME section" on this.

**Review by drgrice1 — APPROVED** [review]
This looks good.  Clearly there is more room for improvement on this, but this is a start.

**Comment by pstaabp** [comment]
I'll take a look at `podchecker`.  It would be nice to do some more cleanup of some of the macro files--I'm also noticing that a lot should be deprecated.

**Comment by pstaabp** [comment]
More cleanup after running `podchecker`.  This is a good first round of edits.

**Comment by drgrice1** [comment]
I have carefully checked that the only thing that is changed in this pull request is POD, and that no code changes were done.  The POD for all macros (except PG.pl and those in deprecated) passes all `podchecker` tests.  Furthermore, the generated html POD using the webwork2 `bin/dev_scripts/generate-ww-pg-pod.pl` script looks considerably better for many of the macros, and there is now POD for some macros that didn't have it before.

So I will merge this now.

**Pull request merged**